aws/resource_aws_waf_rate_based_rule.go

```diff
@@ -70,15 +57,9 @@ func resourceAwsWafRateBasedRule() *schema.Resource {
 				Required: true,
 			},
 			"rate_limit": &schema.Schema{
-				Type:     schema.TypeInt,
-				Required: true,
-				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
-					value := v.(int)
-					if value < 2000 {
-						errors = append(errors, fmt.Errorf("%q cannot be less than 2000", k))
-					}
-					return
-				},
+				Type:         schema.TypeInt,
+				Required:     true,
+				ValidateFunc: validation.IntAtMost(2000),
```

Comment:
Shouldn't this be `validation.IntAtLeast(2000)`?

Reply:
😓